ext/RMagick/extconf.rb

```diff
@@ -113,6 +113,8 @@ def configure_compile_options
         have_library('X11')
 
       end
+
+      $CFLAGS << ' -std=c99'
```

Comment:
@Watson1978 `c99` seems like a pretty old standard. Are we able to use `c11` instead?

Reply:
Looks like there's also `c18`, but maybe that's too recent.